scheduler/util.go

```diff
@@ -811,7 +811,8 @@ func updateNonTerminalAllocsToLost(plan *structs.Plan, tainted map[string]*struc
 
 		// If the scheduler has marked it as stop already but the alloc wasn't
 		// terminal on the client change the status to lost.
```

Comment:
Non-blocker but something to consider: we could update this comment to account for the additional `evict` state or maybe even just delete the comment because it repeats (slightly more readably?) the same information in the function's docstring.